WebApi.Tests/IdentityRepositoryServiceFake.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Core.Domain;
+using Core.DomainServices;
+using Core.DomainServices.Helper;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+
+namespace WebApi.Tests
+{
+    class IdentityRepositoryServiceFake : IIdentityRepository
+    {
+        private readonly AuthHelper _authHelper;
+        public IdentityRepositoryServiceFake(IConfiguration configuration)
+        {
+            _authHelper = new AuthHelper(configuration);
+        }
+        async public Task<JwtSecurityToken> Create(User User, string Password)
+        {
+
+            if (Password == null) { throw new Exception("Password not given"); }
+            if (User.Email == null) { throw new Exception("The specified string is not in the form required for an e-mail address."); }
```

Comment:
Dit is niet goed engels. Ik weet niet goed wat hier de bedoeling is dus verzin maar iets leuk :).